terraform/service_cleanup_export.tf

```diff
@@ -0,0 +1,122 @@
+# Copyright 2020 Google LLC
```

Comment:
Moved these to separate files. Will likely follow-up with a PR to break apart the other services for readability.

Reply:
Good call- I had been thinking about doing the same.